core/ui/EditTemplate/title.tid

```diff
@@ -35,10 +49,14 @@ tags: $:/tags/EditTemplate
 
 <$checkbox tiddler="$:/config/RelinkOnRename" field="text" checked="yes" unchecked="no" default="no"> {{$:/language/EditTemplate/Title/Relink/Prompt}}</$checkbox>
 
```

Comment:
This is where we should include the new warning prompt string mentioned above. It should be preceded with a simple button showing the arrow icon, which won't need the transclusion of `<<lingo TiddlerInfo/References/Caption>>`.